dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/OracleStatsQuery.java

```diff
@@ -30,6 +30,8 @@
 @ParametersAreNonnullByDefault
 public abstract class OracleStatsQuery {
 
+  abstract int queriedDays();
```

Comment:
I would suggest to keep the time duration in the `Duration` type and convert it to days only at the latest possible moment.

Reply:
Done